INSTALL.md

```diff
@@ -1,246 +1,187 @@
+You'll need the following dependencies to compile LibreSprite:
 
-2. Skia back-end (Windows, Mac OS X): You will need a compiled version
-   of the Skia library. Please check the details about
-   [how to build Skia](#building-skia-dependency) on your platform.
+* freetype (FTL): https://www.freetype.org
+* giflib (MIT): http://giflib.sourceforge.net
+* gtest (BSD-3-Clause): https://github.com/google/googletest
+* libjpeg-turbo (BSD-3-Clause IJG Zlib): https://www.libjpeg-turbo.org
+* libpng (Libpng): http://www.libpng.org
+* libwebp (BSD-3-Clause) - https://developers.google.com/speed/webp
+* libx11 (XFree86): https://xorg.freedesktop.org
+* libxcursor (MIT): https://xorg.freedesktop.org
+* pixman (MIT): http://www.pixman.org
+* tinyxml (Zlib): http://www.grinninglizard.com/tinyxml
+* zlib (Zlib): https://www.zlib.net
 
-## Windows dependencies
+### Linux dependencies
 
-Run
+To install the required dependencies on Debian/Ubuntu, run:
+
+    sudo apt-get install cmake g++ libfreetype6-dev libgif-dev libgtest-dev libjpeg-dev libpixman-1-dev libpng-dev libtinyxml-dev libx11-dev libxcursor-dev ninja-build zlib1g-dev
+
+### Windows dependencies
+
+To install the required dependencies with vcpkg, run:
 
     vcpkg install freetype giflib gtest libjpeg-turbo libpng libwebp pixman tinyxml zlib --triplet x64-windows
     
- Beware: `--triplet x64-windows` is only necessary for a 64 architecture
- build.
+Beware: `--triplet x64-windows` is only necessary for a 64 architecture
+build.
 
-After that you have to choose the back-end:
+### MacOS dependencies
 
-1. If you choose the Allegro back-end, you can jump directly to the
-   [Compiling](#compiling) section.
+On MacOS you will need Mac OS X 10.11 SDK and Xcode 7.3 (older versions
+might work).
 
-2. If you choose the Skia back-end, you will need to
-   [compile Skia](#skia-on-windows) before and then continue in the
-   [Compiling](#compiling) section. Remember to check the
-   [Windows details](#windows-details) section to know how to call
-   `cmake` correctly.
+## Compiling
 
-## Mac OS X dependencies
+To compile LibreSprite, run the following commands:
 
-On OS X you will need Mac OS X 10.11 SDK and Xcode 7.3 (maybe older
-versions might work).
+    cd LibreSprite
+    mkdir build
+    cd build
 
-Also, you must compile [Skia](#skia-on-mac-os-x) before starting with
-the [compilation](#compiling).
+Then following the platform-specific instructions for compiling below.
 
-## Linux dependencies
+After you've finished compiling, the executable file will be placed in the
+`build/bin` directory. If you want to build a fresh copy of LibreSprite, remove
+the `build` directory and recompile.
 
-Debian/Ubuntu:
+Once you've finished compiling, you can install LibreSprite with the following
+command:
 
-    sudo apt-get install cmake g++ libfreetype6-dev libgif-dev libgtest-dev libjpeg-dev libpixman-1-dev libpng-dev libtinyxml-dev libx11-dev libxcursor-dev ninja-build zlib1g-dev
+    ninja install
```

Comment:
is this info needed? Maybe I'm looking at that as someone who only uses these branches for testing. Maybe a separate section?

Reply:
Well, it says you *can* install, not that you have to. It was present in the original instructions.

Reply:
I'd create a section in the end then, like "Installation" and put it there. So there is a chronological order